apps/website/src/app/components/Footer.tsx

```diff
@@ -10,21 +11,54 @@ export const Footer = () => {
         &copy; NameHash Labs. All Rights Reserved
       </p>
       <div className="flex flex-row justify-between gap-3">
-        <a href={"https://twitter.com/NamehashLabs"}>
-          <TwitterIcon />
+        <a
+          href="https://twitter.com/NamehashLabs"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <TwitterIcon
+            className={
+              "gt_mobile:transition gt_mobile:hover:fill-black text-[#AFAFAF] fill-current transition-all duration-200"
```

Comment:
I'm still not sure why we need `gt_mobile` breakpoint defined but let's discuss that later 😄

Reply:
I think it has to be this way. Default break points don't fit newest Iphone's and Samsung's screen sizes, ex. IP 14 Pro Max has 430px screen-width and I guess majority of the users is likely to use devices like this. Maybe we could extend mobile version to the smallest default breakpoint, but yeah it's a discussion for later